examples/mongodb/1.8/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mongo:3.6
+FROM mongo:4.2
```

Comment:
Any particular reason for not using 4.4?

Reply:
Yes, as  menitoned during the call, 4.4 requires intra replica set authentication (using key). When I crack this nut for 5.0 I can create 4.4 and backport it here. Now I belive it is important to get working image out ASAP

Reply:
OK, got it.

Reply:
@gunnarmorling Here is the switch directly to 5.0 - https://github.com/debezium/docker-images/pull/271